packages/dripsy/src/core/types-v2/sx.ts

```diff
@@ -64,13 +65,16 @@ type NativeSx = {
 
 // #region Sx
 
+type NonStyleableSxProperties = keyof NativeSx | keyof WebShadowSx
+
 type StyleableSxProperties = Exclude<
-  | Exclude<keyof ThemeUICSSProperties, 'textShadow' | 'boxShadow' | 'variant'>
+  | Exclude<keyof ThemeUICSSProperties, 'variant'>
   | keyof ViewStyle
   | keyof TextStyle
   | keyof ImageStyle
+  | keyof Omit<Aliases, Object.SelectKeys<Aliases, NonStyleableSxProperties>>
```

Comment:
Sorry for the late review. This could just just use `Pick` right?

It's a bit tricky to review all this and know it's working haha TS is so hard to look at in that way

Reply:
@nandorojo Do you mean to replace `Object.SelectKeys<Aliases, NonStyleableSxProperties>` with `keyof Pick<Aliases, NonStyleableSxProperties>>`?
This will not work, because `Pick` selects by keys, while `Object.SelectKeys` selects by values, which is required for the map of aliases.

As the alternative solution, [the `aliases` object](https://github.com/nandorojo/dripsy/blob/master/packages/dripsy/src/core/css/scales.ts#L3) can be split into 2 ones: styleable and non-styleable, and then this line can be replaced with:
```suggestion
  | keyof StyleableAliases
```